docs/pages/kubernetes-access/getting-started.mdx

```diff
@@ -0,0 +1,352 @@
+teleport-cluster-6c9b88fd8f-glmhf   1/1     Running   0          127m
+```
+
+## Step 3/3 SSO for Kubernetes
+
+We are going to setup Github connector for OSS and Okta for Enterpise version.
+
+<Tabs>
+<TabItem label="Open Source">
+
+Save the file below as `github.yaml` and update the fields. You will need to set up
+[Github OAuth 2.0 Connector](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/) app.
+Any member with the team `admin` in the organization `octocats` will be able to assume a builtin role `admin`.
+ 
+```yaml
+kind: github
+version: v3
+metadata:
+  # connector name that will be used with `tsh --auth=github login`
+  name: github
+spec:
+  # client ID of Github OAuth app
+  client_id: client-id
+  # client secret of Github OAuth app
+  client_secret: client-secret
+  # This name will be shown on UI login screen
+  display: Github
+  # Change tele.example.com to your domain name
+  redirect_url: https://tele.example.com:443/v1/webapi/github/callback
+  # Map github teams to teleport roles
+  teams_to_logins:
+    - organization: octocats # Github organization name
+      team: admin           # Github team name within that organization
+      # map github admin team to Teleport's "admin" role
+      logins: ["admins"]
+```
+</TabItem>
+<TabItem label="Enterprise">
+
+Follow [SAML Okta Guide](../enterprise/sso/ssh-okta.mdx#configure-okta) to create a SAML app.
+Check out [OIDC guides](../enterprise/sso/oidc.mdx#identity-providers) for OpenID Connect apps.
+Save the file below as `okta.yaml` and update the `acs` field.
+Any member in Okta group `okta-admin` will assume a builtin role `admin`.
+
+```yaml
+kind: saml
+version: v2
+metadata:
+  name: okta
+spec:
+  acs: https://tele.example.com/v1/webapi/saml/acs
+  attributes_to_roles:
+  - {name: "groups", value: "okta-admin", roles: ["admin"]}
+  entity_descriptor: |
+    <?xml !!! Make sure to shift all lines in XML descriptor 
+    with 4 spaces, otherwise things will not work
+```
+</TabItem>
+</Tabs>
+
+To create a connector, we are going to run Teleport's admin tool `tctl` from the pod.
+
+<Tabs>
+<TabItem label="Open Source">
+```bash
+# To create a Github connector, we are going to run Teleport's admin tool tctl from the pod.
+$ POD=$(kubectl get po -l app=teleport-cluster -o jsonpath='{.items[0].metadata.name}')
+
+$ kubectl exec -i ${POD?} tctl -- create -f < github.yaml
+authentication connector "github" has been created
+```
+</TabItem>
+<TabItem label="Enterprise">
+```bash
+# To create an Okta connector, we are going to run Teleport's admin tool tctl from the pod.
+$ POD=$(kubectl get po -l app=teleport-cluster-ent -o jsonpath='{.items[0].metadata.name}')
+
+$ kubectl exec -i ${POD?} tctl -- create -f < okta.yaml
+authentication connector 'okta' has been created
+```
+</TabItem>
+</Tabs>
+
+Try `tsh login` with Github user. I am using a custom `KUBECONFIG` to prevent override
+of the default one in case if there is a problem on the first try.
```

Comment:
```suggestion
Try `tsh login` with Github user. I am using a custom `KUBECONFIG` to prevent overwriting
the default one in case there is a problem.
```